src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-decoration: none;
+    color: #060b35;
```

Comment:
What I expect is to use a variable here for example (but you need to use it through all your code)
```suggestion
    color: $primary_text_color;
```

and a file with variables like this 

![image](https://github.com/mate-academy/layout_product-cards/assets/46566190/3a8e5f28-8983-453f-b1f4-a03db1e47293)